src/services/services.ts

```diff
@@ -1381,8 +1381,8 @@ module ts {
     enum MatchKind {
         none = 0,
         exact = 1,
-        substring = 2,
-        prefix = 3
+        prefix = 2,
+        substring = 3,

```

Comment:
Because they're part of our public JSON Api.  Changing them would break our clients.